ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__419-aws-ovn.yaml

```diff
@@ -135,8 +135,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- always_run: false
```

Comment:
@creydr Do you know why these are removed? (and if that's OK? )

Reply:
Not sure, but I guess it's not needed on cron jobs :thinking:

Reply:
It's probably coming from this change https://github.com/openshift-knative/hack/commit/d0953d0e3a8c6157e33cb5c1dbde2219a7280ad6#diff-8c104313544b2c9a8413d2cbb7a399e56793ffab4a3e554d5ac2243a5a74f1c8L33   ,

Reply:
yeah, I assume it doesn't matter for cron jobs.